pytx3/pytx3/cli/fetch2.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import requests
+import typing as t
+from .. import TE
```

Comment:
For my own edification, what's the difference between `from . import XXX` and `from .. import XXX`

Reply:
Basic version is:

`from . import XXX` means from the directory that houses this file, import the module defined in the file XXX. So here it would import the file XXX.py from `pytx3/pytx3/cli`

`from .. import xxx` means from the parent directory of the directory that houses this file, import the module defined in the file XXX. So here it would import the file XXX.py from `pytx3/pytx3`

`from ..dataset import XXX` means from the module dataset that inside the parent directory of the directory of this file, import the class or symbol XXX. So here it would import the class or symbol XXX that is defined in the file `pytx3/pytx3/dataset.py`

There's other details I'm probably missing, but that's a basic gist of its use here.

Reply:
genius. thank you